ui/app/helpers/mountable-secret-engines.js

```diff
@@ -1,84 +1,90 @@
-import { helper as buildHelper } from '@ember/component/helper';
+import { helper as buildHelper } from "@ember/component/helper";
 
 const MOUNTABLE_SECRET_ENGINES = [
   {
-    displayName: 'Active Directory',
-    value: 'ad',
-    type: 'ad',
-    category: 'cloud',
+    displayName: "Active Directory",
+    value: "ad",
+    type: "ad",
+    category: "cloud"
   },
   {
-    displayName: 'AWS',
-    value: 'aws',
-    type: 'aws',
-    category: 'cloud',
+    displayName: "AWS",
+    value: "aws",
+    type: "aws",
+    category: "cloud"
   },
   {
-    displayName: 'Azure',
-    value: 'azure',
-    type: 'azure',
-    category: 'cloud',
+    displayName: "Azure",
+    value: "azure",
+    type: "azure",
+    category: "cloud"
   },
   {
-    displayName: 'Consul',
-    value: 'consul',
-    type: 'consul',
-    category: 'infra',
+    displayName: "Consul",
+    value: "consul",
+    type: "consul",
+    category: "infra"
   },
   {
-    displayName: 'Databases',
-    value: 'database',
-    type: 'database',
-    category: 'infra',
+    displayName: "Databases",
+    value: "database",
+    type: "database",
+    category: "infra"
   },
   {
-    displayName: 'Google Cloud',
-    value: 'gcp',
-    type: 'gcp',
-    category: 'cloud',
+    displayName: "Google Cloud",
+    value: "gcp",
+    type: "gcp",
+    category: "cloud"
   },
   {
-    displayName: 'KV',
-    value: 'kv',
-    type: 'kv',
-    category: 'generic',
+    displayName: "AliCloud",
```

Comment:
Can we move this up so that the list is alpha sorted by displayName? Kinda arbitrary but an easy way to do it that doesn't show preference for any of them.

Reply:
yes! i like that idea.